torch/optim/optimizer.py

```diff
@@ -754,9 +758,11 @@ def register_load_state_dict_pre_hook(
         hook: Callable[["Optimizer", StateDict], Optional[StateDict]],
         prepend: bool = False,
     ) -> RemovableHandle:
-        r"""Register a load_state_dict pre-hook which will be called before
-        :meth:`~torch.optim.Optimizer.load_state_dict` is called. It should have the
-        following signature::
+        r"""Register a load_state_dict pre-hook.
```

Comment:
why is this change necessary? The first and second separated line here is a better first line together.